cmd/erasure-object.go

```diff
@@ -862,28 +863,35 @@ func (er erasureObjects) DeleteObjects(ctx context.Context, bucket string, objec
 
 	versions := make([]FileInfo, len(objects))
 	for i := range objects {
+		modTime := opts.MTime
+		if opts.MTime.IsZero() {
+			modTime = UTCNow()
+		}
+		uuid := mustGetUUID()
+		if opts.VersionID != "" {
+			uuid = opts.VersionID
+		}
```

Comment:
style nit: to keep the conditional assignment of `uuid` similar to `modTime` (above).
```suggestion
		uuid := opts.VersionID
		if uuid == "" {
			uuid = mustGetUUID()
		}
```

Reply:
done